todo_app/app.py

```diff
@@ -1,15 +1,81 @@
-from flask import Flask
 
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.data.session_items import get_item, get_items, add_item, clear_items, clear_item, update_item_status, sort_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
-def index():
-    return 'Hello World!'
+@app.route('/<op_message>')
+def index(op_message = ""):
+    # Get the list of items from the pre-loaded session file and sort by status prior to loading
+    items = sort_items(get_items())
+
+    # Get the version out from the file
+    with open('TODO_APP/VERSION.txt','rt') as ver_file:
+        ver = ver_file.read()
```

Comment:
You can delete this and the VERSION.txt file.

If you want to report on the version of your app, that's already in the pyproject.toml file.

You can access it with 
```
import pkg_resources
my_version = pkg_resources.get_distribution('todo-app').version
```

Reply:
Hi Hugh, this is interesting. I can give that a try. What is the method for updating this version within the file, is it to open as text and manually edit or is there an api to call?